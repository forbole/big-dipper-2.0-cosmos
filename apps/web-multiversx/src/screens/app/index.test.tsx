import type { Router } from 'next/router';
import renderer from 'react-test-renderer';
import App from '@/screens/app';

// ==================================
// unit tests
// ==================================
describe('screen: _app', () => {
  it('matches snapshot', () => {
    const component = renderer.create(
      <App router={{} as Router} Component={() => <div id="component" />} pageProps={{}} />
    );
    const tree = component?.toJSON();
    expect(tree).toMatchSnapshot();
  });

  afterEach(() => {
    jest.clearAllMocks();
  });
});
