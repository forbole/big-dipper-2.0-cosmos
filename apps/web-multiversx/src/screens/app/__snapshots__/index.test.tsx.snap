// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`screen: _app matches snapshot 1`] = `
[
  <div
    id="component"
  />,
  <div
    className="Toastify"
  />,
]
`;
